build/templates/_converters.py.mako

```diff
@@ -237,7 +275,8 @@ def convert_to_nitclk_session_number(item):
     raise TypeError('Unsupported type for nitclk session: {}'.format(type(item)))
 
 
-def convert_to_nitclk_session_number_list(item_list):
+# The unused parameter is required for case B520 in codegen, which expects a two-parameter converter.
```

Comment:
we've renumbered the cases serveral times, I feel like this will accidentally get stale at some point. I'm not sure if we're better off removing it though because it certainly is useful.

Do we have converters that only take one parameter? Maybe we should standardize the interface and always take two parameters. If you agree then we should do that prep-work in its own PR.

Reply:
Discussed with Marcos offline. We decided that we can standardize on all converters to have one parameter - created a PR - #1302.

Reply:
Don't have `unused` parameter anymore :)